lib/async_experiments.rb

```diff
@@ -37,7 +37,17 @@ def self.get_experiment_data(experiment_name)
         extra: extra_entries,
         changed: changed.map(&:last),
       }
-    }
+    end
+  end
+
+  def self.get_experiment_exceptions(experiment_name)
+    Sidekiq.redis do |redis|
+      enumerator = redis.scan_each(
+        match: "experiments:#{experiment_name}:exceptions:*"
+      )
+      retrieve = -> (key) { redis.get(key) }
+      enumerator.map(&retrieve).compact
+    end
```

Comment:
might be nice to extract some of this logic that's shared with `get_experiment_data`